src/lib.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright 2015-2017 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Parity Brain Wallet Generator.
+
+#![warn(missing_docs)]
+
+#[macro_use]
+extern crate lazy_static;
+
+extern crate itertools;
+extern crate rand;
+
+use itertools::Itertools;
+use rand::{Rng, OsRng};
+
+/// Generate a string which is a random phrase of a number of lowercase words.
+///
+/// `words` is the number of words, chosen from a dictionary of 7,530. An value of
+/// 12 gives 155 bits of entropy (almost saturating address space); 20 gives 258 bits
+/// which is enough to saturate 32-byte key space
+pub fn random_phrase(words: usize) -> String {
+	lazy_static! {
+		static ref WORDS: Vec<String> = String::from_utf8_lossy(include_bytes!("../res/wordlist.txt"))
```

Comment:
Overkill because of adding an extra dependency? Parsing a `Vec<String>` from a JSON array is [a one-liner](http://play.integer32.com/?gist=bb6dba32b05a8a4d0b057197fcc8110b&version=undefined).